code/modules/mob/living/simple_animal/abnormality/aleph/staining_rose.dm

```diff
@@ -142,4 +161,68 @@
 		L.physiology.black_mod *= 0.5
 		L.physiology.pale_mod *= 0.5
 
+
+//SCHISMATIC
+//Decrease defenses of heretics.
+/datum/status_effect/schismatic
+	id = "schismatic"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1		//Lasts until the rose dies
+	alert_type = /atom/movable/screen/alert/status_effect/schismatic
+
+/atom/movable/screen/alert/status_effect/schismatic
+	name = "Schismatic"
+	desc = "You have ruined the sanctity between the rose and it's chosen and have been punished."
+
+/datum/status_effect/schismatic/on_apply()
+	. = ..()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/L = owner
+		L.physiology.red_mod /= 0.2
```

Comment:
This is x5 damage taken

Reply:
Sorry, It was supposed to be x1.2 or /0.8 but my brain fucked it up